### PR TITLE
added crud operation on to-do

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> used LocalStorage to persist the user data even after reloading..
-> used useContext to access the user data on different .
-> user can create,update and delete the task
-> task can also be marked as completed or uncompleted.

### CODE REVIEW

File: src/Components/Modal/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    } else {
+      setUser_Data({ type: "ADD", paylod: curr_todo });
+    }
+    toggle_modal();
+    reset_modal();
+  };
+  return (
+    <div className="pop_up">
+      <div className="pop_up_background"></div>
+      <div className="pop_up_input fnt-2 pad-1-5">
```
Comment (ArjunGTX) [inline src/Components/Modal/Modal.jsx:24]: why there are 2 patterns in class names? isn't it better to stick to one?

File: src/Pages/Home/Home.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+.todo-body {
+  background-color: white;
+  flex: 1 1 auto;
+  border-radius: 30px 30px 0px 0px;
+}
+
+.todo-header {
+  padding: 0 5rem;
+  /* line-height: 1.2; */
```
Comment (ArjunGTX) [inline src/Pages/Home/Home.css]: remove commented code

File: src/Pages/Home/Home.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+.todo-text {
+  height: 100%;
+  margin-right: auto;
+  overflow-y: scroll;
+  text-overflow: ellipsis;
+}
+
+.todo-text::-webkit-scrollbar {
```
Comment (ArjunGTX) [inline src/Pages/Home/Home.css:57]: Why is scrollbar being hidden?

File: src/Pages/Home/Home.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
+import "./Home.css";
+import { useState } from "react";
+import { useUserTodo } from "../../Context/useUserTodo";
+import { no_of_pending_task } from "../../Utils/no_of_pending_task";
+import { Modal } from "../../Components";
+import { useGetLocalData } from "../../Hooks/useGetLocalData";
+
+export const Home = () => {
+  useGetLocalData();
+  const { user_data, setUser_Data } = useUserTodo();
```
Comment (ArjunGTX) [inline src/Pages/Home/Home.js]: either follow `lower_snake_case` or `camelCase` don't mix up both. ~~setUser_Data~~ `set_user_data` or `setUserData`. `camelCase` is convention

File: src/Pages/Home/Home.css (hunk 3 above)
Comment (ashishnnnnn) [reply src/Pages/Home/Home.css:57]: Made scrollbar visible with custom colour and style

File: src/Pages/Home/Home.css (hunk 2 above)
Comment (ashishnnnnn) [reply src/Pages/Home/Home.css]: Done

File: src/Pages/Home/Home.js (hunk 4 above)
Comment (ashishnnnnn) [reply src/Pages/Home/Home.js]: Changed it..

**Review by AmishMishra11 — APPROVED** [review]
Good code implementation

File: src/Pages/Home/Home.css (hunk 3 above)
Comment (ArjunGTX) [reply src/Pages/Home/Home.css:57]: Great

**Review by ArjunGTX — APPROVED** [review]

**Pull request merged**